controllers/state_manager.go

```diff
@@ -592,9 +593,11 @@ func getRuntimeString(node corev1.Node) (gpuv1.Runtime, error) {
 	case strings.HasPrefix(runtimeVer, "cri-o"):
 		runtime = gpuv1.CRIO
 	default:
-		return "", fmt.Errorf("runtime not recognized: %s", runtimeVer)
+		return "", "", fmt.Errorf("runtime not recognized: %s", runtimeVer)
 	}
-	return runtime, nil
+	version := strings.SplitAfter(runtimeVer, "//")[1]
+	vVersion := strings.Join([]string{"v", strings.TrimPrefix(version, "v")}, "")
```

Comment:
I think the following is easier to read?
```suggestion
	vVersion := "v" + strings.TrimPrefix(version, "v")
```